assets/css/src/components-plugin/_listing.scss

```diff
@@ -97,6 +97,7 @@
 }
 
 .mailpoet-listing-pages-num {
+  font-weight: normal;
```

Comment:
It's `font-weight: 600` in the design, normal is 400. Why are you using it?

Reply:
You're right, I guess I mixed it up with `font-stretch`. I removed that commit, so it's back to were it was.